internal/microsoft_edge.py

```diff
@@ -126,7 +126,7 @@ def get_driver(self, task):
                     shutil.copy(src, extension_dir)
             except Exception:
                 pass
-        capabilities['extensionPaths'] = [extension_dir]
+        capabilities['ms:extensionPaths'] = [extension_dir]
```

Comment:
Is there harm in setting both? If not, I'd feel better for backward compatibility if it was set for both keys.

Reply:
Setting of both keys didn't cause any errors with the new Webdriver. So I don't see any problem.